src/PKSim.Core/Mappers/ProcessToProcessBuilderMapper.cs

```diff
@@ -286,26 +316,60 @@ public TransporterMoleculeContainer ActiveTransportFrom(CompoundProcess compound
          return transporterMoleculeContainer;
       }
 
-      // private void updateTransporterTagsFor(IndividualTransporter transporter, ITransportBuilder transporterBuilder, string simulationProcessName)
-      // {
-      //    //if one organ was specified already, no need to create the list of not tags!
-      //    var allOrgans = transporter.AllOrgansWhereProcessTakesPlace(simulationProcessName).ToList();
-      //    var allMatchTags = transporterBuilder.SourceCriteria.OfType<MatchTagCondition>()
-      //       .Select(x => x.Tag);
-      //
-      //    if (allMatchTags.Intersect(allOrgans).Any())
-      //       return;
-      //
-      //    foreach (var organName in transporter.AllOrgansWhereProcessDoesNotTakePlace(simulationProcessName))
-      //    {
-      //       transporterBuilder.SourceCriteria.Add(new NotMatchTagCondition(organName));
-      //    }
-      // }
-
-      private ITransportBuilder activeTransportFrom(CompoundProcess process, string individualProcessName, IFormulaCache formulaCache)
+      private IReadOnlyCollection<InducedProcess> allInducedProcessesFor(IndividualTransporter transporter, Individual individual)
+      {
+         var inducedProcessCache = new Cache<string, InducedProcess>(x => x.Name, x => null);
+         var allTransporterContainers = individual.AllMoleculeContainersFor<TransporterExpressionContainer>(transporter)
+            .Where(x => x.TransportDirection != TransportDirectionId.None).ToList();
+
+         //Global containers surrogate do not have a logical container associated
+         var allOrganNames = new HashSet<string>(allTransporterContainers.Select(x => x.LogicalContainerName).Where(x => !string.IsNullOrEmpty(x)));
+
+         foreach (var transporterContainer in allTransporterContainers)
+         {
+            var allTransportTemplates = _transportTemplateRepository.All()
```

Comment:
This is a new table in the db as discussed @Yuri05 
Adding this multiple transport directions really helped a lot to simplify the logic